camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusHourlySweeper.java

```diff
@@ -142,6 +142,10 @@ protected void runCollectorForTopicDir(FileSystem fs, String topic, Path topicSo
     return executorService.submit(new KafkaCollectorRunner(jobName, props, errorMessages, topic));
   }
 
+  private static void createTimeStampFileInFolder(FileSystem fs, Path folder, long timeStamp) throws IOException {
```

Comment:
Please write the create timestamp to a more generic file (Java properties or Json). We should not need to create a separate file if we want to add more metadata,